src/analyzer.ts

```diff
@@ -117,16 +117,29 @@ export class Analyzer {
   }
 
   /**
+   * Returns `true` if the provided resolved URL can be loaded.
```

Comment:
Although, I'm looking forward to updating `polymer-bundler` to say `new InMemoryOverlayUrlLoader(opts.analyzer)`